lib/policyEvaluator/evaluator.ts

```diff
@@ -324,7 +324,10 @@ export function evaluateAllPolicies(
     requestContext: RequestContext,
```

Comment:
to avoid breaking compatibility, may be better not to touch the prototype of this function : i.e. make the change to the body in a function with a new name, and transform the older function in a simpler wrapper
→ this way existing code using it is not affected, it can decide when they "upgrade" to the new API, and we don't introduce coupling or limit upgrade paths
→ ideally we could even mark the 'old' method as deprecated, though I am not sure what is the impact currently: depending on build flags it could probably fail the build...

Something like this:

```ts
/* @deprecated Upgrade to evaluateAllPoliciesV2
 */
export function evaluateAllPolicies(
    requestContext: RequestContext,
    allPolicies: any[],
    log: Logger,
): string {
    return evaluateAllPoliciesV2(requestContext, allPolicies, log).verdict;
}

export function evaluateAllPoliciesV2(
    requestContext: RequestContext,
    allPolicies: any[],
    log: Logger,
): {
    verdict: string;
    isImplicit: boolean;
} {
   ...
}
```